### ISSUE TITLE
CSS combining not working with inline svg rgb definition

### ISSUE THREAD

**Issue by jacobro** [contao/core#8788]
Contao 3.5.28. I use inline svg inside my css. I noticed, that after combining, contao adds a quote inside rgb color definition.
My css is:
```css
background-image: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg'><path stroke='rgb(255, 255, 255)'></svg>");
```
after combining it looks as shown below
```css
background-image: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg'><path stroke='rgb(255, 255, 255")'></svg>");
```
Combiner adds needless double quote just before rgb ending bracket

**Comment by leofeyer** [comment]
What happens if you use single quotes for the image and double quotes for the attributes?

```css
background-image: url('data:image/svg+xml;charset=utf8,<svg xmlns="http://www.w3.org/2000/svg"><path stroke="rgb(255, 255, 255)"></svg>');
```

**Comment by jacobro** [comment]
It outputs sth much more weird than before, as it still adds double quote before ending bracket and also changes first single quote after `url(` to double quote. I think combiner "thinks" that rgb ending bracket is the end of url definition.
```css
background-image: url("data:image/svg+xml;charset=utf8,<svg xmlns="http://www.w3.org/2000/svg"><path stroke="rgb(255, 255, 255")"></svg>');
```

**Comment by leofeyer** [comment]
Fixed in a2be3cb372ae6761ac2deb3469157046ae2b4eb7.
